SpywareFilter/sections/cookies.txt

```diff
@@ -34,4 +33,142 @@
 $cookie=_parsely_visitor
 $cookie=_parsely_session
 ! Tealium Universal tag
-$cookie=utag_main
\ No newline at end of file
+$cookie=utag_main
+! Google - https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage
+$cookie=_gid
+$cookie=_ga
+$cookie=/^_gat_tracker\d+/
+$cookie=_ga_devsite
+$cookie=/^_gat_/
+$cookie=_gat
+$cookie=AMP_TOKEN
+$cookie=/^_gac_/
+$cookie=/^_ga_/
+$cookie=/^_dc_gtm_UA-/
+$cookie=/^CaosGtag_ga/
+$cookie=_gaexp
+$cookie=_gaexp_rc
+$cookie=utma
```

Comment:
Should be `__utma` (other `utm*` cookies as well)

Reply:
Right, I will change it.
By the way, this website `https://www.qarson.fr/` is using `utma`, so maybe we should add both rules, a least for `utma`.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/114750837-b7b77780-9d54-11eb-9147-bbbe87520742.png)

</details>

Reply:
Already there is `__utma` and some other rules which starts with `__` below - https://github.com/AdguardTeam/AdguardFilters/pull/80418/files#diff-502809f1020ded4a26ff4ad467ffe53d0e3f42db352794e3537d2910604ea95bR64

Reply:
> Right, I will change it.
> By the way, this website https://www.qarson.fr/ is using utma, so maybe we should add both rules, a least for utma.

Seems to be a custom cookie they're setting.